pkg/util/gpg.go

```diff
@@ -154,7 +192,10 @@ func getPublicKey(id string) ([]byte, error) {
 	return nil, errNoSuchPublicKey
 }
 
-var errNoSuchPublicKey = errors.New("no such public key")
+var (
+	noPublicKeyFoundRx = regexp.MustCompile(`no(t)?.*found`)
```

Comment:
The parentheses around the `t` are not required: `t?` is the same thing as `(t)?`. Furthermore, since you're explicitly looking for either "no" or "not", I assume you want a word boundary match directly afterwards as well to avoid matching "no" as a part of another larger word?
```suggestion
	noPublicKeyFoundRx = regexp.MustCompile(`not?\b.*found`)
```